atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -135,6 +136,10 @@ def set_help(self, extra, worker_metadatas):
                         # They are all None
                         extra['image']['help'] = None
 
+    def set_brew_versions(self, extra, worker_metadatas):
+        build_id, brew_version = self.workflow.postbuild_results.get(PulpPullPlugin.key)
```

Comment:
Update: the worker build sets the `help_file` in its annotations. Then in `koji_import`, the annotations are fetched again from the worker build (build.get_annotations() vs .get_annotations()). So yes, for help_file, it bypasses the need to make `orchestrate_build` aware of it. I'm not sure if this is actually better though. Seems to me, the initial idea was to have annotations being fetched from one place, `orchestrate_build`.